src/style.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: Arial, sans-serif;
+  text-align: center;
+}
+
+h1 {
+  color: #333;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  margin-top: 20px;
+}
+
+.left-section,
+.right-section {
+  flex: 1;
+}
+
+.header-container {
```

Comment:
- [ ] Please fix the below style issues to match the wireframe design. 
  - Please make sure the `Leaderboard` heading is aligned on the left side.
  - Please make sure the `Recent Score` heading is aligned on the left above the score section.
  - Please make sure the refresh button is aligned on the right above the score section


### Your design

![image](https://github.com/Dorcas126/Leaderboard-project/assets/83803414/3410c857-6f04-49f0-971e-ca46993bcdf8)

### Wireframe design

![image](https://github.com/Dorcas126/Leaderboard-project/assets/83803414/8de5faa5-0c33-46be-92f0-0c271d30192d)